### PR TITLE
Reduce supply chain attacks vectors

### PR DESCRIPTION
**Pull request by tuyuribr**
# PR Details

For some "smaller" dependencies we should hard-code specific versions to make it harder to make a supply chain attack

## Types of changes

- [x] Dependency change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,11 +30,11 @@
   "homepage": "https://wwebjs.dev/",
   "dependencies": {
     "@pedroslopez/moduleraid": "^5.0.2",
-    "fluent-ffmpeg": "^2.1.2",
-    "jsqr": "^1.3.1",
+    "fluent-ffmpeg": "2.1.2",
+    "jsqr": "1.3.1",
```
Comment (alechkos) [inline package.json:34]:
```suggestion
```
@tuyuribr 
`jsqr` already is not in use, should we get rid of it? 
Was added in [#26](https://github.com/pedroslopez/whatsapp-web.js/commit/bea1ebf48035f52b71b145214eac0dfd6576b113) and then its usage was removed in [#68](https://github.com/pedroslopez/whatsapp-web.js/commit/5c9e76e23f52418eaa3062e33194eb916c9565ef)

Comment (alechkos) [inline package.json:34]:
```suggestion
```
@tuyuribr 
`jsqr` already is not in use, should we get rid of it? 
Was added in [#26](https://github.com/pedroslopez/whatsapp-web.js/commit/bea1ebf48035f52b71b145214eac0dfd6576b113) and then its usage was removed in [#889](https://github.com/pedroslopez/whatsapp-web.js/commit/0d55d408850ef5214fde751d8b55af42fd70aef1#diff-b880f47f381d68dcd7473fb094acb874cd50b1140cdf2bfd7a368fa509738073)

Comment (tuyuribr) [reply package.json:34]: We should ahhaha

**Review by alechkos — APPROVED** [review]

**Review by shirser121 — APPROVED** [review]
LGTM, even we should update our dependencies

**Pull request merged**